src/DatePicker/DateDisplay.js

```diff
@@ -5,33 +5,41 @@ import SlideInTransitionGroup from '../internal/SlideIn';
 function getStyles(props, context, state) {
   const {datePicker} = context.muiTheme;
   const {selectedYear} = state;
+  const isLandscape = props.mode === 'landscape';
 
   const styles = {
     root: {
+      width: isLandscape ? 125 : 270,
+      height: isLandscape ? 290 : 'auto',
+      float: isLandscape ? 'left' : 'none',
+      fontWeight: 'bolder',
```

Comment:
can we use a specific font weight? we may not have a `bolder` version loaded depending on what is being inherited